GAD-7.json

```diff
@@ -1,14 +1,21 @@
 {
-  "status": "draft",
+  "code": [
+    {
+      "code": "69737-5",
```

Comment:
@mcjustin does the `code` here uniquely identifies this questionnaire?

Reply:
@achen2401  it does uniquely identify it at LOINC. but I don't think we'll use it as a canonical reference. LOINC won't have all of our questionnaires, and their codes can be hard to identify. Also, I like our existing pattern of referring to questionnaires by strings which are more recognizable (eg "epic26", "mini-cog").